src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportUnitTest.java

```diff
@@ -41,8 +41,10 @@
                 {"Sample1\tfile1\t"},           // extra tab
                 {"Sample1\nfile"},              // newline instead of tab
                 {"\t"},                         // only tab
-                {"Sample1 file1"},              // non-tab whitespace
-                {"Sample 1\tfile1"},            // white space in a token
+                {"Sample1 file1"},              // 1 column
+                {" name\tfile1"},               // preceding whitespace
+                {"name \tfile1"},               // trailing whitespace
+                {"name\tfile1 "},               // trailing whitespace second column
```

Comment:
In addition to these negative test cases, you should add some positive test cases for `GenomicsDBImport.loadSampleNameMapFile()` to prove that that method can handle sample names with internal spaces.